taskcluster/fenix_taskgraph/target_tasks.py

```diff
@@ -4,36 +4,23 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
-import os
-import re
-
 from taskgraph.target_tasks import _target_task, filter_for_tasks_for
 
 
-BETA_SEMVER = re.compile(r'^v\d+\.\d+\.\d+-beta\.\d+$')
-PRODUCTION_SEMVER = re.compile(r'^v\d+\.\d+\.\d+(-rc\.\d+)?$')
-
-
 @_target_task('default')
```

Comment:
Given that this matches the upstream definition, you don't need to define it here.

Reply:
I thought it did too, but it doesn't really. I cannot use the upstream definition because of https://hg.mozilla.org/ci/taskgraph/file/02bc397b581bb679fe872a2deea96f0c5cad8589/src/taskgraph/target_tasks.py#l35